src/test/java/seedu/address/logic/AutoCompleteTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.logic;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import seedu.address.logic.trie.CommandTrie;
+import seedu.address.logic.trie.Trie;
+
+/**
+ * Test Class for autocomplete function
+ */
+public class AutoCompleteTest {
+
+    private Trie commandTrie;
+
+    @Before
+    public void setup() {
+        commandTrie = new CommandTrie();
+    }
+
+    @Test
+    public void testAutoComplete() {
+        assert commandTrie.attemptAutoComplete("a").equals("add");
+        assert commandTrie.attemptAutoComplete("e").equals("e");
+        assert commandTrie.attemptAutoComplete("ed").equals("edit");
+        assert commandTrie.attemptAutoComplete("ex").equals("exit");
+        assert commandTrie.attemptAutoComplete("H").equals("H");
+        assert commandTrie.attemptAutoComplete("He").equals("help");
```

Comment:
is that all the commands that can perform autocomplete?

Reply:
Instead of testing all the possible commands, I instead tested the two possible cases - a: there is only one possible command for the given entry and b: there are multiple or no possible commands for a given entry. Should I test all the possible options instead?

Reply:
It is recommended to test minimally all the commands that you think it should work (especially for autocomplete as it is one of the features that are most susceptible to modification). 

This prevents future modification or introduction of new commands breaking your feature, and this is the main purpose of having test cases.